kessler/time_utils.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+def getCcsdsTimeFormat(timeString):
+    '''
+    Adapted by Andrew Ng, 18/3/2022.
+    Original MATLAB source code found at: https://github.com/nasa/CARA_Analysis_Tools/blob/master/two-dimension_Pc/Main/TransformationCode/TimeTransformations/getCcsdsTimeFormat.m
+    The CCSDS time format is required to be of the general form
+    yyyy-[mm-dd|ddd]THH:MM:SS[.F*][Z]
+    (1) The date and time fields are separated by a "T".
+    (2) The date field has a four digit year followed by either a two digit 
+        month and two digit day, or a three digit day-of-year.  
+    (3) The year, month, day, and day-of-year fields are separated by a dash.
+    (4) The hours, minutes and seconds fields are each two digits separated 
+        by colons.
+    (5) The fraction of seconds is optional and can have any number of
+        digits.
+    (6) If a fraction of seconds is provided, it is separated from the two
+        digit seconds by a period.
+    (7) The time string can end with an optional "Z" time zone indicator
```

Comment:
it would be nice if in the docstrings you could add Args and Returns. For example:

```
    Adapted by Andrew Ng, 18/3/2022.
    Original MATLAB source code found at: https://github.com/nasa/CARA_Analysis_Tools/blob/master/two-dimension_Pc/Main/TransformationCode/TimeTransformations/getCcsdsTimeFormat.m
    The CCSDS time format is required to be of the general form
    yyyy-[mm-dd|ddd]THH:MM:SS[.F*][Z]
    (1) The date and time fields are separated by a "T".
    (2) The date field has a four digit year followed by either a two digit 
        month and two digit day, or a three digit day-of-year.  
    (3) The year, month, day, and day-of-year fields are separated by a dash.
    (4) The hours, minutes and seconds fields are each two digits separated 
        by colons.
    (5) The fraction of seconds is optional and can have any number of
        digits.
    (6) If a fraction of seconds is provided, it is separated from the two
        digit seconds by a period.
    (7) The time string can end with an optional "Z" time zone indicator
    
    Args:
         -time_string (``str``): input description
  
  Returns:
         - timeFormat (``type``): output description
```